<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BeDistinctiveOrRelatedActivity</class>
 <widget class="QWidget" name="BeDistinctiveOrRelatedActivity">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>737</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QFrame" name="framez">
     <property name="styleSheet">
      <string notr="true">QFrame#framez {
    border: 3px dashed white;
}
</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="promptLabel">
        <property name="styleSheet">
         <string notr="true">font: 300 15pt &quot;Bookman Old Style&quot;;
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>Select all flags that match the displayed flags.</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QPushButton" name="hintButton">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 300 14pt &quot;Bookman Old Style&quot;;
background-color: transparent;
border: 1px solid black;</string>
          </property>
          <property name="text">
           <string>Show Hint</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="hint">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 300 10pt &quot;Bookman Old Style&quot;;
color: rgb(0, 0, 0);</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="displayFlag1">
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>150</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>300</width>
            <height>150</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="displayFlag2">
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>150</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>300</width>
            <height>150</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="flag1Score">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="flag1">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>300</width>
              <height>150</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>900</width>
              <height>450</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="flag2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>300</width>
              <height>150</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>900</width>
              <height>450</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="flag2Score">
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="flag4Score">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="flag4">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>300</width>
              <height>150</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>900</width>
              <height>450</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="flag3">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>300</width>
              <height>150</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>900</width>
              <height>450</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="flag3Score">
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="next">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 300 14pt &quot;Bookman Old Style&quot;;
background-color: transparent;
border: 1px solid black;</string>
          </property>
          <property name="text">
           <string>Check Answer</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="progressBar">
          <property name="value">
           <number>24</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
